radio/src/gui/colorlcd/model_outputs.cpp

```diff
@@ -216,6 +216,13 @@ void ModelOutputsPage::build(FormWindow * window, int8_t focusChannel)
   grid.spacer(PAGE_PADDING);
   grid.setLabelWidth(66);
 
+  new TextButton(window, grid.getLineSlot(), STR_ADD_ALL_TRIMS_TO_SUBTRIMS,
```

Comment:
Wouldn't this actually be Move? It moves the trims to the offsets, aka subtrim, thus resetting the trims. I know where the confusion comes from, so lets not dwell on that ;)

Looks like a screen refresh is needed after this happens as well.

Reply:
> Wouldn't this actually be Move? It moves the trims to the offsets, aka subtrim, thus resetting the trims. I know where the confusion comes from, so lets not dwell on that ;)
> 
> Looks like a screen refresh is needed after this happens as well.

i have done the screen reset, and I am testing on simulator26.  I think it is actually Add and not Move.  If you move the trims, press the button, and then move the trims again, the numbers keep going up.  So it is Adding not Moving.

Let me know?

Reply:
I made the change to the button so it says move, but I think it should be changed back to add.  If you like it this way then merge, otherwise I will change it back to Add